README.md

```diff
@@ -8,20 +8,131 @@ Use the R package [remotes](https://cran.r-project.org/web/packages/remotes/inde
 
 ```R
 remotes::install_github('VEuPathDB/plot.data')
+
+# or to install a specific version
+remotes::install_github('VEuPathDB/plot.data', 'v1.2.3')
 ```
+<br/>
 
 ## Usage
 
+All `plot.data` functions require at least the following arguments:
+1. A data frame or data table with columns corresponding to variables and rows to samples.
```

Comment:
idk if samples is the best word here. records? observations?

Reply:
Covered all bases -- changed it to "...columns corresponding to variables and rows to records (for example, observations, samples, etc.)"